packages/app/src/components/SecurityTab/SecurityTab.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { EmbeddedScaffolderWorkflow } from '@backstage/plugin-scaffolder';
+import { Box } from '@material-ui/core';
+
+const ReviewWrapper = () => {
+  return (
+    <Box display="flex" alignItems="center" flexDirection="column">
+      <h1>This is a different wrapper for the review page</h1>
+    </Box>
+  );
+};
+
+/*
+ * This is an exmaple component that uses the <EmbeddedScaffolderWorkflow />
+ */
+export function SecurityTab(): JSX.Element | null {
```

Comment:
Examples & documentation on how to adopt this plugin would be great, but I'm not sure we should add this to the `packages/app`. Suggestion would be to exclude it from this PR & maybe create a follow up PR in case we want this in the `app` (@Rugvip)

Reply:
reverted